### PR TITLE
feat: add flag to force start finish log

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

feat

**Did you add tests for your changes?**
wip

**If relevant, did you update the documentation?**
NA

**Summary**
Add a env var to force compilation log

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Review by alexander-akait — COMMENTED** [review]
For watch too

**Comment by anshumanv** [comment]
@alexander-akait ready

File: test/build/start-finish-force-log/start-finish-force-log.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            env: { WEBPACK_CLI_START_FINISH_FORCE_LOG: true },
+        });
+        expect(exitCode).toBe(0);
+        expect(stderr).toContain("Compiler 'Gojou' starting...");
+        expect(stderr).toContain("Compiler 'Satoru' starting...");
+        expect(stderr).toContain("Compiler 'Gojou' finished");
+        expect(stderr).toContain("Compiler 'Satoru' finished");
+        expect(stdout).toBeTruthy();
+    });
+});
```
Comment (alexander-akait) [inline test/build/start-finish-force-log/start-finish-force-log.test.js:46]: Can we test here `compiled successfully`

**Review by alexander-akait — APPROVED** [review]

File: test/build/start-finish-force-log/start-finish-force-log.test.js (hunk 1 above)
Comment (anshumanv) [reply test/build/start-finish-force-log/start-finish-force-log.test.js:50]: done

**Comment by anshumanv** [comment]
@webpack/cli-team

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
Need docs

**Pull request merged**